src/rockstor/storageadmin/static/storageadmin/js/templates/common/navbar.jst

```diff
@@ -40,6 +40,7 @@
       <li><a href="#config-backup"><i class="glyphicon glyphicon-floppy-save"></i> Config Backups</a></li>
       <li><a href="#email"><i class="fa fa-envelope"></i> Email Alerts</a></li>
       <li><a href="#logsmanager"><i class="fa fa-gears"></i> Logs Manager</a></li>
+      <li><a href="#shell"><i class="fa fa-terminal"></i> System Shell</a></li>
```

Comment:
Hi @schakrava , small icon on navbar via 5a015b4
If ok for you we can procede :smiley: